### DISCUSSION TITLE
Is there any way to get an on-screen VU meter like PiCorePlayer has?

### DISCUSSION THREAD

**Discussion by apastuszak** [rern/rAudio#991] [Feature Requests]
I love VU meters. I'd love there to be an on-screen VU meter while a song is playing when you have a screen attached to the Pi.

**Comment by rern** [comment]
Settings > Playback - Options: VU meter

**Reply by apastuszak** [reply]
The VU meter doesn't move.

Is there any way to display a left and right channel VU meter?
